static/index.js

```diff
@@ -51,6 +51,24 @@ function appendOutput(content, className) {
   output.appendChild(element);
 }
 
+function highlightDebugPath(className) {
```

Comment:
Can you do this when inserting the content into the DOM instead of inserting, querying it, and then updating it please 🙏